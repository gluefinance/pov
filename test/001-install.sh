#!/bin/sh
dropdb test 2> /dev/null
createdb test
psql test -f /crypt/postgresql-8.4.6/contrib/pgcrypto/pgcrypto.sql
cd sql
psql -f uninstall.sql test 2> /dev/null
psql -f install.sql test
cd ..
psql -f test/001-install.sql test
