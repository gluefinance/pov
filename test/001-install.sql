BEGIN;
SET client_min_messages = 'debug';
SELECT * FROM snapshot();
CREATE FUNCTION myfunc() RETURNS VOID AS $$ $$ LANGUAGE sql;
\df myfunc
SELECT * FROM snapshot();
SELECT * FROM snapshot(1);
\df myfunc
ROLLBACK;
