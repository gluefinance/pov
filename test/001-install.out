SET
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
DROP ROLE
DROP ROLE
CREATE ROLE
CREATE ROLE
CREATE SCHEMA
CREATE LANGUAGE
SET
BEGIN
SET
CREATE FUNCTION
CREATE FUNCTION
SET
CREATE TABLE
CREATE TABLE
CREATE TABLE
CREATE SEQUENCE
CREATE TABLE
CREATE INDEX
INSERT 0 1
INSERT 0 1
INSERT 0 1
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE FUNCTION
CREATE VIEW
CREATE VIEW
CREATE VIEW
SET
CREATE VIEW
GRANT
GRANT
GRANT
GRANT
GRANT
GRANT
COMMIT
BEGIN
SET
 _snapshotid |               _revisionid                
-------------+------------------------------------------
           1 | 871ff0b837436ea1357814cdaa35a2013c63bd44
(1 row)

CREATE FUNCTION
                         List of functions
 Schema |  Name  | Result data type | Argument data types |  Type  
--------+--------+------------------+---------------------+--------
 public | myfunc | void             |                     | normal
(1 row)

 _snapshotid |               _revisionid                
-------------+------------------------------------------
           2 | 1e96cbf3da6aa5d648c837e6e910dfbe7f3eae18
(1 row)

 _snapshotid |               _revisionid                
-------------+------------------------------------------
 3           | 871ff0b837436ea1357814cdaa35a2013c63bd44
(1 row)

                       List of functions
 Schema | Name | Result data type | Argument data types | Type 
--------+------+------------------+---------------------+------
(0 rows)

ROLLBACK
